### ISSUE TITLE
GGMax charactorcreatorplus REQUEST/improvement

### ISSUE THREAD

**Issue by andypmillard** [TheGameCreators/GameGuruRepo#3657]
Can the customisation of clothes, hair etc not be made available across all genders, zombies and humans. Why you ask? Well for example the only fireman is a zombie. Opening up these items would increase the scope of characters many fold.

**Comment by Tam51** [comment]
I agree with you on this one. It would also be nice if we could import our own custom characters that are rigged and animated. I think improvements need to be made to make it easier to import characters without having to do all sorts of things to make them work.

**Comment by synchromesh62** [comment]
I dont think its clothing as such they are body parts for the characters gender
You would have a tough guy with a female jacket and Boobs :)

**Comment by toonyman2020** [comment]
**Requests:**

New and Improve Character Creation tool - Character Creation is actually terrible as you can only pick characters already made and maybe customize it a little and buggy (i'm trying not to use the word buggy as is in early development). so I'm requesting to changing or creating something better for what is in todays standard of character creation! such as morphing body parts like in daz studio, reallusion, vroid studio, cyberpunk 2077, VAM or VAMX etc. even blender has give us character poses, this new character could allow or unique character creation and that you can also import your own including your own nude model characters. there a new market for this kind a stuff to explore and give players something new and unique of experience! FPS games are just to overrate and overly done more than i can ever count!

this way it's not through max but the end user. simply put a guy like me we imply just want to make our game and not what g,g max wants us to make! creative freedom is the key word here! I want to be able make my games that i want and not the games you want me to make due to creative freedom limitations and without coding or creating assets as is advertise! is why i invested money and time into this and hope you guys hold up your end up of the deal! otherwise I can't give this a positive review! I just want a fair chance in creating the games I envision or dream of is why many are here! game engine can't have limitations those blender, unity or unreal have limitations? that what makes it a professional game engine this is currently not at the moment. i'm no coder nor will i ever be at this time otherwise i wouldn't even bother coming here. I just want to get straight into making games without all these other distractions and this what I hope you guys can deliver as advertise. an almost hassle free game engine no coding or asset creation is what this was advertise as. as an artist I just can't have limitations here hope you understand how insulting that really is.
